src/wallet.md

```diff
@@ -0,0 +1,121 @@
+### solana-wallet CLI
+
+The [solana crate](https://crates.io/crates/solana) is distributed with a command-line interface tool
+
+Usage:
+
+```sh
+$ solana-wallet [common-options] [command] [command-specific options]
+```
+
+`common-options` include:
+* `--fee xyz` - Transaction fee (0 by default)
+* `--output file` - Write the raw Transaction to a file instead of sending it
+
+`command` variants:
+* `pay`
+* `cancel`
+* `send-signature`
+* `send-timestamp`
+
```

Comment:
Additional available commands:
- address
- airdrop
- balance
- confirm
- deploy
- get-transaction-count

Reply:
```suggestion
#### Get Pubkey
```sh
// Command
$ solana-wallet address

// Return
<PUBKEY>
```
```

Reply:
```suggestion
#### Airdrop Tokens
```sh
// Command
$ solana-wallet airdrop 123

// Return
"Your balance is: 123"
```

Reply:
```suggestion
#### Get Balance
```sh
// Command
$ solana-wallet balance

// Return
"Your balance is: 123"
```

Reply:
```suggestion
#### Confirm Transaction
```sh
// Command
$ solana-wallet confirm <TX_SIGNATURE>

// Return
"Confirmed" / "Not found"
```

Reply:
```suggestion
#### Deploy program
```sh
// Command
$ solana-wallet deploy <PATH>
// Return
<PROGRAM_ID>
```

Reply:
```suggestion
#### Get Transaction Count
```sh
// Command
$ solana-wallet get-transaction-count

// Return
"123"
```